docs/setup/overview.md

```diff
@@ -41,14 +39,31 @@ This option should be preferred if you have access to the controllers of the mac
 Data is signed on the shop-floor still close to the data-source (the closer to the data-source, the better for security).
 
 **Integration:**<br />
-You can use your existing connectors, the Agent is integrated at the message bus layer.
+You can use your existing connectors, the Agent is integrated at the message bus layer (e.g. [MQTT](../provide_data/iot/mqtt.md), [OPC-UA](../provide_data/iot/opc-ua.md), [Beckhoff ADS](../provide_data/iot/ads.md),...).
 
 ### DSK Agent Embedded {#dsk-agent-embedded}
 
-The DSK Agent Embedded reads data directly from sensors and uses a dedicated hardware security module for signing the data-streams. This is available as either a C implementation or as a dedicated hardware module in the form of our Sensor Security Module (SSM).
+The DSK Agent Embedded is our solution designed to read data directly from interfaces/sensors on the hardware level. It uses a dedicated hardware security module for signing the data-streams. 
+The DSK Agent Embedded is available as either a C implementation or as a dedicated hardware module.
+For the hardware module we offer two options:
+* [OEM Module](https://tributech.io/blog/oem-module)  
+We offer two variants, the OEM Module itself which can be directly integrated into your hardware design and an Arduino Shield for prototyping.
+The shield can be used with all popular hardware development platforms like Arduino Uno, STM32 Nucelo, Infineon XMC Relax Kit and more.  
+![OEM Module](./img/OEM_Module.jpg)  
+An IoT device based on the OEM Module typically consists of three parts. Part 1: the Tributech OEM Module which takes care of connectivity, data management, configurations, updates and security. Part 2: the application-specific microcontroller which takes care of data pre-processing and business logic. Part 3: the sensors and interfaces needed for the application. 
+![OEM Board](./img/OEM_Board.jpg)  
+On the device side, the OEM module provides a UART interface for exchanging configurations, providing telemetry data and performing updates. This allows developers to focus on firmware development for data pre-processing and sensor/interface integration, as all complexity is abstracted via an easy-to-use interface.<br/><br/>
+Overview highlighting the role of the OEM module in the overall DSK:
+![OEM Board](./img/OEM_Architecture.jpg)
+
+* [Sensor Security Module (SSM)](https://tributech.io/blog/sensor-security-module)  
```

Comment:
i thought we agreed on removing info about ssm, please check with PO

Reply:
I discussed with PO and we won't publish SSM doc (I left it commented as desired by PO).